samples/common/Kconfig

```diff
@@ -60,6 +60,25 @@ config GOLIOTH_SAMPLE_DHCP_BIND
 	help
 	  Obtain an IPv4 address from the DHCP server.
 
+config GOLIOTH_SAMPLE_FW_VERSION
```

Comment:
After NCS bump to version "post v2.4.99-dev2" there is `CONFIG_MCUBOOT_IMGTOOL_SIGN_VERSION` used in NCS (which is the same symbol as in vanilla Zephyr). This means that main (I guess) reason why `GOLIOTH_SAMPLE_FW_VERSION` was introduced is no longer valid.

Additionally there is a way to define application version with Zephyr infrastructure, by placing `VERSION` file in application directory. See https://github.com/zephyrproject-rtos/example-application/commit/2c85d9224fca21fe6e370895c089a6642a9505ea for example usage. Not sure this is viable option for our SDK, since we just provide information on how to set version from commandline in order to quickly test that feature, not permanently because we release application.

Since there are already 2 ways (see above) how (application) version can be set, I don't see good enough reason to introduce another way.